app/views/goldencobra/admin/widgets/_article_widgets_sidebar.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_for(resource, url: connect_with_articles_admin_widget_path, method: :post) do |f| %>
+  Nur wenn mindestens eine Niederlassung ausgewählt ist, ist die Geolokalisierung des Widgets aktiv!<br/><br/>
```

Comment:
Hier eigentlich nichts kundenspezifisches reinschreiben.  Das Widget hier hat ja nicht mit irgendwelchen Niederlassungen zu tun. Das Partial kann in QPB überschrieben werden, um einen speziellen Text einzufügen, würde ich sagen.

Reply:
der Hinweis ist zudem ein Copy Paste Fehler